validation/src/main/scala/hmda/validation/rules/ts/syntactical/S010.scala

```diff
@@ -1,7 +1,7 @@
 package hmda.validation.rules.ts.syntactical
 
 import hmda.model.fi.ts.TransmittalSheet
-import hmda.validation.dsl.{ Result, CommonDsl }
+import hmda.validation.dsl.{ CommonDsl, Result }
```

Comment:
Looks like you cleaned up imports here and elsewhere. Do we want to have some sort of cleanup step as part of the build/pre-commit process, so we don't get spurious diffs? (Not as part of this PR, and perhaps a discussion for its own issue, but I wanted to say something while thinking about it.)